packages/salesforcedx-vscode-lwc/test/vscode-integration/jsconfig.test.ts

```diff
@@ -51,6 +45,12 @@ describe('jsconfig Test Suite', () => {
     }
   });
 
+  beforeEach(async () => {
```

Comment:
Strange that the `beforeEach()` needs to come after the `afterEach()` (the order shouldn't matter).  Could you add a comment, letting the reader know why they are in that order, and warn them to not move them?  I could see someone seeing this one day and changing the order, thinking they are cleaning up the code.